spacy/lookups.py

```diff
@@ -6,7 +6,7 @@
 
 from .errors import Errors
 from .util import SimpleFrozenDict, ensure_path, registry, load_language_data
-from .strings import get_string_id
+from .strings import get_string_id  # type: ignore
```

Comment:
Can you fix the types rather than ignoring this?

Reply:
My initial attempts to get that import to type check seemed to point to a `mypy` limitation (`get_string_id` is a `cpdef` function, which apparently isn't recognized by the former), but I'll rummage around a little more to see if there's another workaround.

Reply:
I think you should just add it to `strings.pyi`.